src/index.js

```diff
@@ -239,11 +251,11 @@ class Firestore {
     });
 
     /**
+     * A client pool to distribute requests over multiple GAPIC clients.
```

Comment:
Perhaps make it clear that we do this to work around the limit in the GAPIC client.

Reply:
Done.